linera-views/src/reentrant_collection_view.rs

```diff
@@ -670,7 +670,7 @@ where
     }
 }
 
-/// A view that supports accessing a collection of views of the same kind, indexed by key,
+/// A view that supports accessing a collection of views of the same kind, indexed by keys,
```

Comment:
(Here I also thought "by key" would be more common.)

Reply:
Well, there are more than one key, so the plural is probably correct.